src/classes/Frames.ts

```diff
@@ -1,10 +1,13 @@
-import { ErrorType } from "../types";
+import { ErrorType, UserVTOFrames } from "../types";
 import Auth from "./Auth";
 
-export const getVTOFrames = async ({ sku }): Promise<ErrorType | void | {}> => {
+export const getVTOFrames = async ({ sku }): Promise<ErrorType | UserVTOFrames> => {
   try {
     const userProfile = await Auth.getUserProfile();
 
+    console.log("userProfile: ", userProfile);
+    console.log("sku: ", sku);
+
     const userVTOFrames = userProfile?.vto?.['1']?.[`${sku}`]?.frames || [];
```

Comment:
Can we make this not be statically defined? It needs to be the dynamic value provided in the sdk init.